tests/conftest.py

```diff
@@ -1,9 +1,18 @@
 import numpy as np
+import pytest
 from pytest import fixture
 
 import pyvista
 from pyvista import examples
 
+import vtk
+
+
+@fixture()
+def vtk9():
+    if not vtk.vtkVersion().GetVTKMajorVersion() >= 9:
+        pytest.skip('Skipping VTK9+ only test')
```

Comment:
I think using a fixture here is a bit confusing, but I like where you're headed with not rewriting the skip function.  Try this instead:

```suggestion
skip_no_vtk9 = pytest.mark.skipif(not vtk.vtkVersion().GetVTKMajorVersion() >= 9, reason="Requires VTK9+")
```

Follow this by replacing the `vtk9` fixture with
```py
@skip_no_vtk9
def test_some_function_that_needs_vtk9():
    # some test that needs vtk9...
```